test/Table_Tests/src/In_Memory/Split_Tokenize_Spec.enso

```diff
@@ -0,0 +1,174 @@
+from Standard.Base import all
+
+import Standard.Base.Data.Text.Case_Sensitivity.Case_Sensitivity
+import Standard.Test.Extensions
+
+from Standard.Table import Table
+from Standard.Table.Errors import Invalid_Value_Type, Column_Count_Exceeded, Duplicate_Output_Column_Names, Missing_Input_Columns
+from Standard.Test import Test, Test_Suite, Problems
+from project.Util import all
+
+spec =
+    Test.group "Table.split" <|
+        Test.specify "can do split_to_columns" <|
+            cols = [["foo", [0, 1, 2]], ["bar", ["abc", "cbdbef", "ghbijbu"]]]
+            t = Table.new cols
+            expected_rows = [[0, "a", "c", Nothing], [1, "c", "d", "ef"], [2, "gh", "ij", "u"]]
+            expected = Table.from_rows ["foo", "bar_0", "bar_1", "bar_2"] expected_rows
+            t2 = t.split_to_columns "bar" "b"
+            t2.should_equal_verbose expected
+
+        Test.specify "can do split_to_rows" <|
+            cols = [["foo", [0, 1, 2]], ["bar", ["abc", "cbdbef", "ghbijbu"]]]
+            t = Table.new cols
+            expected_rows = [[0, "a"], [0, "c"], [1, "c"], [1, "d"], [1, "ef"], [2, "gh"], [2, "ij"], [2, "u"]]
+            expected = Table.from_rows ["foo", "bar"] expected_rows
+            t2 = t.split_to_rows "bar" "b"
+            t2.should_equal_verbose expected
+
+    Test.group "Table.tokenize" <|
+        Test.specify "can do tokenize_to_columns" <|
+            cols = [["foo", [0, 1, 2]], ["bar", ["a12b34r5", "23", "2r4r55"]]]
+            t = Table.new cols
+            expected_rows = [[0, "12", "34", "5"], [1, "23", Nothing, Nothing], [2, "2", "4", "55"]]
+            expected = Table.from_rows ["foo", "bar_0", "bar_1", "bar_2"] expected_rows
+            t2 = t.tokenize_to_columns "bar" "\d+"
+            t2.should_equal_verbose expected
+
+        Test.specify "can do tokenize_to_rows" <|
+            cols = [["foo", [0, 1, 2]], ["bar", ["a12b34r5", "23", "2r4r55"]]]
+            t = Table.new cols
+            expected_rows = [[0, "12"], [0, "34"], [0, "5"], [1, "23"], [2, "2"], [2, "4"], [2, "55"]]
+            expected = Table.from_rows ["foo", "bar"] expected_rows
+            t2 = t.tokenize_to_rows "bar" "\d+"
+            t2.should_equal_verbose expected
```

Comment:
And also, I think it would be good to have tests for both split/tokenize where some of the input cells are `Nothing` (since a Text column can still contain 'nulls').

If these tests already exist - sorry but I could not find them, and I think it's important to have them.